scripts/run_mteb_french.py

```diff
@@ -0,0 +1,78 @@
+"""Example script for benchmarking all datasets constituting the MTEB French leaderboard & average scores"""
+
+import logging
+
+from sentence_transformers import SentenceTransformer
+
+from mteb import MTEB
+
+logging.basicConfig(level=logging.INFO)
+
+logger = logging.getLogger("main")
+
+TASK_LIST_CLASSIFICATION = [
+    "AmazonReviewsClassification",
+    "MasakhaNEWSClassification",
+    "MassiveIntentClassification",
+    "MassiveScenarioClassification",
+    "MTOPDomainClassification",
+    "MTOPIntentClassification",
+]
+
+TASK_LIST_CLUSTERING = [
+    "AlloProfClusteringP2P",
+    "AlloProfClusteringS2S",
+    "HALClusteringS2S",
+    "MasakhaNEWSClusteringP2P",
+    "MasakhaNEWSClusteringS2S",
+    "MLSUMClusteringP2P",
+    "MLSUMClusteringS2S",
+]
+
+TASK_LIST_PAIR_CLASSIFICATION = [
+    "OpusparcusPC",
+]
+
+TASK_LIST_RERANKING = [
+    "SyntecReranking",
+    "AlloprofReranking"
+]
+
+TASK_LIST_RETRIEVAL = [
+    "AlloprofRetrieval", 
+    "BSARDRetrieval", 
+    "HagridRetrieval",
+    "SyntecRetrieval"
+]
+
+TASK_LIST_STS = [
+    "SummEvalFr",
+    "STSBenchmarkMultilingualSTS",
+    "STS22",
+    "SICKFr"
+]
+
+TASK_LIST_BITEXTMINING = [
+    "DiaBLaBitextMining",
+    "FloresBitextMining",
```

Comment:
I think Tatoeba & BUCC also have French in case you want to add them

Reply:
Added

Reply:
Done.